main.go

```diff
@@ -143,7 +144,7 @@ func (m model) View() string {
 	// Display words per minute when finished
 	if len(m.typed) >= len(m.text) {
 		s += fmt.Sprintf(bold(`WPM: %.2f
-    `), words/m.end.Sub(m.start).Minutes(),
+    `), (float64(m.score)/5.)/(m.end.Sub(m.start).Minutes()),
```

Comment:
Can you make the 5. a constant up top like charsPerWord

Reply:
snm 096b519